### PR TITLE
Feature/even more improvements

### LINKED ISSUE #57: caseSensitive bei subCommands von tag
Die subCommands von `tag`  müssen derzeit immer **klein** geschrieben werden. Es wäre sehr praktisch, vorallem für die Handy-Nutzer, wenn die Groß- und Kleinschreibung dort einfach ignoriert werden würde.

LG Random

PS: bei den Namen eines tags wäre dies auch gut

### PR DESCRIPTION
**Pull request by DRSchlaubi**
This requires `feature/improvements`

Fixes #57

### CODE REVIEW

File: src/main/kotlin/com/github/seliba/devcordbot/command/context/Context.kt [hunk 1]
```diff
@@ -133,7 +139,11 @@ data class Context(
      */
     fun hasModerator(): Boolean = hasPermission(Permission.MODERATOR)
 
-    private fun hasPermission(permission: Permission) =
-        commandClient.permissionHandler.isCovered(permission, member) == PermissionState.ACCEPTED
+    fun hasPermission(permission: Permission) =
+        commandClient.permissionHandler.isCovered(permission, member, devCordUser) == PermissionState.ACCEPTED
+
+    private fun notifyCommndHanler(action: MessageAction) = action.map {
```
Comment (jkoenig134) [inline src/main/kotlin/com/github/seliba/devcordbot/command/context/Context.kt]: typo -> NotifyCommandHandler

File: .idea/compiler.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="CompilerConfiguration">
-    <bytecodeTargetLevel target="13" />
+    <bytecodeTargetLevel target="14" />
   </component>
 </project>
```
Comment (jkoenig134) [inline .idea/compiler.xml:6]: newline 😆

File: src/main/kotlin/com/github/devcordde/devcordbot/event/DevCordMessageReceivedEvents.kt [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                DevCordGuildMessageReceivedEvent(
+                    event.jda,
+                    event.responseNumber,
+                    event.message,
+                    devCordUser
+                )
+            )
+        }
+    }
+}
```
Comment (jkoenig134) [inline src/main/kotlin/com/github/devcordde/devcordbot/event/DevCordMessageReceivedEvents.kt]: Newline

File: .idea/compiler.xml (hunk 2 above)
Comment (DRSchlaubi) [reply .idea/compiler.xml:6]: this file is generated so anytime compiler config changes we would have to add it again

File: src/main/kotlin/com/github/seliba/devcordbot/command/context/Context.kt (hunk 1 above)
Comment (jkoenig134) [reply src/main/kotlin/com/github/seliba/devcordbot/command/context/Context.kt]: TYPOO

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**